code/processes/tailer.q

```diff
@@ -0,0 +1,57 @@
+\d .wdb
+.proc.loadf [getenv[`KDBCODE],"/processes/wdb.q"]
+hdbsettings:(`compression`hdbdir`taildir)!(compression;hdbdir;getenv`KDBTAIL)
```

Comment:
```suggestion
hdbsettings[`taildir]:getenv`KDBTAIL
```

If I'm following correctly, you're really only changing this key in the dict, so no need to overwrite the whole thing